core/templates/dev/head/components/summary_tile/exploration_summary_tile_directive.html

```diff
@@ -28,8 +29,7 @@ <h3 class="activity-title protractor-test-exp-summary-tile-title">
           <span translate="I18N_LIBRARY_NO_OBJECTIVE">
           </span>
         </div>
-
-        <ul layout="row" class="metrics" layout-align="space-between center">
+        <ul ng-if="isCollectionPreviewTile === undefined || !isCollectionPreviewTile" layout="row" class="metrics" layout-align="space-between center">
```

Comment:
While this is not an error in an Angular expression, I think it is odd to have it, since it shouldn't be necessary and would be an error in regular Javascript (to test without typeof).

Angular handles checking undefined and returning true/false internally ([confirmed here](https://plnkr.co/edit/PkkUwJnC8T6bT1TWp3Yg?p=preview)), so these could be simplified.

If undefined was checked for a specific purpose, maybe comment in ExplorationSummaryTileDirective.js should also say undefined rather than 'not null'? https://github.com/oppia/oppia/pull/3197/files#diff-a08595fc5ce9d8f446f95635fa6274e0R38